src/test/assembly/niche-prefer-zero.rs

```diff
@@ -0,0 +1,29 @@
+// Check that niche selection prefers zero.
+// See https://github.com/rust-lang/rust/pull/87794
+// revisions: x64 A64 ppc64le
```

Comment:
You will need to specify the `--target=$TARGET` flags for each of the architectures you're testing via `[$REVISION]compile-flags`. `revision` by itself does not have any meaning whatsoever.

In order to enable this, you may will also need to build for `#[no_core]` like some other assembly tests do. Or you could make this a `only-x86` test instead.

Reply:
Going for `only-x86` instead.
I don't even now how to test the others